index.html

```diff
@@ -3881,10 +3881,21 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="image">
+			<rdef>image</rdef>
+			<div class="role-description">
+				<p>A container for a collection of <a>elements</a> that form an image. See synonym <rref>img</rref>.</p>
+				<div class="note" id="role-image-note-image">
+					<h5>Note regarding the ARIA 1.3 <code>image</code> role.</h5>
+					<p>In ARIA 1.3, the working group introduced <code>image</code> as a synonym to the <rref>img</rref> role. This was done to mitigate author confusion and errors due to the role being an outlier compared to all other roles, which are complete words or concatenations of complete words.</p>
```

Comment:
I think we can emphasize the newness and streamline statement of intent as follows.

```suggestion
					<p>The <code>image</code> role was added to ARIA in version 1.3. As a synonym of the ARIA 1.0 <rref>img</rref> role,
the <code>image</code> role improves syntactic consistency with the names of other roles, which are complete words or concatenations of complete words.</p>
```

Reply:
i'm fine with this, but i'd want to revise the wording of the note for`none` and `presentation` so we don't have two different wordings for the same sort of message in the spec.